### PR TITLE
Removed the usage of INFRA lists, in favor of ordered sets everywhere.

### PR DESCRIPTION
**Pull request by iherman**
This is a companion PR to issue #518, more exactly its [comment](https://github.com/w3c/did-core/issues/518). Submitting this as a PR may speed up processing in case the group agrees with the conclusions in #518...

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/522.html" title="Last updated on Jan 6, 2021, 7:57 AM UTC (fd01d9c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/522/9605a1e...fd01d9c.html" title="Last updated on Jan 6, 2021, 7:57 AM UTC (fd01d9c)">Diff</a>

### CODE REVIEW

**Review by rhiaro — APPROVED** [review]

**Review by TallTed — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -3916,8 +3865,8 @@ <h4>equivalentId</h4>
                 <dl>
                   <dt><dfn>equivalentId</dfn></dt>
                   <dd>
-The value of <code>equivalentId</code> MUST be a <a
-data-cite="INFRA#lists">list</a> where each item in the list is a <a
+The value of <code>equivalentId</code> MUST be an <a
+data-cite="INFRA#ordered-set">ordered set</a> where each item in the list is a <a
```
Comment (msporny) [inline index.html:3869]: This change looks good. I did a scan through the entire spec and we really don't need `list` anywhere... but that's not to say that the core data model shouldn't support it.

**Review by msporny — CHANGES_REQUESTED** [review]
We shouldn't remove support for `list` from the data model... but we should change some of the properties that used `lists` to `ordered sets` because that seems to be the intention for those properties... namely, `equivalentId`.

File: index.html [hunk 2]
```diff
@@ -1201,18 +1201,10 @@ <h1>Data Model</h1>
         </tr>
         <tr>
           <td>
-<a data-cite="INFRA#list">list</a>
```
[the remark this answers is not in the document]
Comment (TallTed) [reply index.html:1204]: Good point. Until this PR, I hadn't considered complete removal of `list` from the data model (about which I agree with @msporny; we should retain this in the data model) before this PR, only changing the few `list` objects we had to `ordered sets` (which I still think is the right call for those properties).

[the remark this answers is not in the document]
Comment (iherman) [reply index.html:1204]: I am fine with that, @msporny. Actually, that means not only lists should remain in the list of official datatypes, but the corresponding consumption/production rules for JSON & CBOR should also stay.

Commit follows...

File: index.html [hunk 3]
```diff
@@ -4210,8 +4210,8 @@ <h2>
 used to communicate this metadata MUST be a <a data-cite="INFRA#maps">map</a>
 of properties. Each property name MUST be a <a
 data-cite="INFRA#string">string</a>. Each property value MUST be a <a
-data-cite="INFRA#string">string</a>, <a data-cite="INFRA#maps">map</a>, <a
-data-cite="INFRA#lists">list</a>, <a data-cite="INFRA#boolean">boolean</a>, or
+data-cite="INFRA#string">string</a>, <a data-cite="INFRA#maps">map</a>, an <a
+data-cite="INFRA#ordered-set">ordered set</a>, <a data-cite="INFRA#boolean">boolean</a>, or
```
Comment (peacekeeper) [inline index.html:4214]: Per @msporny 's comment https://github.com/w3c/did-core/pull/522#discussion_r551029641, probably you want to reinstate "list" here as well.

Comment (iherman) [reply index.html]: Yep. Changed it in https://github.com/w3c/did-core/pull/522/commits/fd01d9c3cda9ec57b78057c97edec0af6cfd992b.

File: index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (iherman) [reply index.html]: Changed in https://github.com/w3c/did-core/pull/522/commits/b9cbbaf9f6346b666b1863ccb1ecb41cac8e2f81.

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Normative, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**